uint8 msgId
bool   anomalyStatus
uint64 timeStamp
